app/views/collections/show.html.slim

```diff
@@ -1,10 +1,26 @@
 .header
   .std-heading
     i.fa-solid.fa-folder-open
-    =< @collection.title
+    =<> @collection.title
+    hr.mb-2
 .row
-  .col-md-12.mt-4
-    .collection-content
+  .col-md-12
+    .collection-subinfo
+      span
+        - if @collection.visibility_level_public?
+          i.fa-solid.fa-lock-open.lock-icon
+          =< t('.visibility.public')
+        - else
+          i.fa-solid.fa-lock.lock-icon
+          =< t('.visibility.private')
+      span
+        i.fa-solid.fa-users
+        - if @collection.users.length == 1
+          =< t('.no_other_user')
+        - else
+          =< t('.num_of_other_users', count: @collection.users.length - 1)
```

Comment:
Is this a deliberate choice for `length` since we need the collection users anyway?
I was just wondering, since further down you call `count`.

Reply:
No, I have not thought about the use of `length` here - The `count` further down was already used before. If I see it correctly `size` should fit them both better.